src/components/Articles/ArticleList.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import ArticleCard from './ArticleCard';
+import './ArticleList.css';
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios.get('https://jesjin-nc-news.onrender.com/api/articles')
+      .then(response => {
+        setArticles(response.data.articles);
+      })
+      .catch(error => {
+        console.error('Error fetching articles:', error);
+      });
+  }, []);
+
+  return (
+    <div className="article-list">
+      {articles.map(article => (
+        <ArticleCard key={article.article_id} article={article} />
+      ))}
+    </div>
+  );
+};
```

Comment:
You can add a loading pattern to this page too, there's an example in the notes. Then you can let the user know that something is happening they just might have to wait for the articles to load :)